openrtb_ext/imp_smartadserver.go

```diff
@@ -0,0 +1,10 @@
+package openrtb_ext
+
+// ExtImpSmartadserver defines the contract for bidrequest.imp[i].ext.smartadserver
+type ExtImpSmartadserver struct {
+	Domain    string `json:"domain"`
```

Comment:
`Domain` doesn't seem to be used by your adapter. I'm not saying we don't want to keep it, I'm just double checking it is not a leftover or something.

Reply:
I missed that. Good eye, @guscarreon.

Reply:
Sorry I forgot it 😕